src/webui/abstractwebapplication.cpp

```diff
@@ -147,20 +147,25 @@ void AbstractWebApplication::removeInactiveSessions()
 
 bool AbstractWebApplication::sessionInitialize()
 {
-    static const QString SID_START = QLatin1String(C_SID) + QLatin1String("=");
-
     if (session_ == 0)
     {
-        QString cookie = request_.headers.value("cookie");
-        //qDebug() << Q_FUNC_INFO << "cookie: " << cookie;
-
-        QString sessionId;
-        int pos = cookie.indexOf(SID_START);
-        if (pos >= 0) {
-            pos += SID_START.length();
-            int end = cookie.indexOf(QRegExp("[,;]"), pos);
-            sessionId = cookie.mid(pos, end >= 0 ? end - pos : end);
-        }
+        const auto parseCookie = [](const Http::Request &request) -> QStringMap
+        {
+            QStringMap ret;
+            const QVector<QStringRef> cookies = request.headers.value(QLatin1String("cookie")).splitRef(';', QString::SkipEmptyParts);
```

Comment:
>AFAIK, cookies can be separated by either a comma or a semicolon.

https://tools.ietf.org/html/rfc6265#section-4.2.1
In RFC 6265:
1. Semicolon is the seperator.
2. Comma cannot be used in `cookie-octet`, but I don't think there is any harm ignoring this rule for server-side...
3. `cookie-value` could be surrounded by double quotes, I will add this part.

>Also this line is too long. Please split it.

OK.